__test__/ExperimentTable.test.js

```diff
@@ -1,11 +1,11 @@
 import React from 'react'
-import { shallow, mount } from 'enzyme'
-import { getRandomInt, TableCellDiv, data, tableHeader } from './TestUtils'
```

Comment:
Actually you don't have to remove the space here, as this is we used when we import the undefault stuff.

Reply:
Okay will remove it. Its the intelliJ formatting which messed it up. Will avoid doing it in future